README.md

```diff
@@ -24,6 +24,46 @@ the following features in mind:
 * Be well-tested and robust to the point that it can be used in large
   projects without exposing unfortunate, disappointing bugs here and there.
 
+## Building
+
+Ormolu builds with `cabal-install` and `ghc-8.4`. For instance:
+
+```bash
+$ nix-shell -p haskell.compiler.ghc844
```

Comment:
I wanted to setup Nix-powered build first and then have Cabal instructions based on that, but well, in the meantime we can just add this.